sql/2021/structured-data/rdfa_prefixes.sql

```diff
@@ -5,7 +5,7 @@ RETURNS ARRAY<STRING>
 LANGUAGE js AS """
   try {
     rendered = JSON.parse(rendered);
-    const prefixRegExp = new RegExp(/(?<ncname>[^:]*):\s+(?<uri>[^\s]*)\s*/gm)
+    const prefixRegExp = new RegExp(/(?<ncname>[^:]*):\\s+(?<uri>[^\\s]*)\\s*/gm)
```

Comment:
```suggestion
    const prefixRegExp = new RegExp(/(?<ncname>[^:]*):\s+(?<uri>[^\s]*)\s*/gm)
```

Reply:
I was confused about this too. Weirdly it wouldn’t rub for me on BigQuery without this. Complained about an illegal escape, which I’d not seen before and made me wonder why it worked for original analyst.

Maybe I had a weird setting enabled. Or was due to lack of `r` in `r”””`. Will give it another look.

Reply:
Yeah it was the lack of `r"""` - TIL!